README.md

```diff
@@ -126,6 +126,11 @@ make any formatting changes to the input, and `1` if Rustfmt would make changes.
 In other modes, Rustfmt will exit with `1` if there was some error during
 formatting (for example a parsing or internal error) and `0` if formatting
 completed without error (whether or not changes were made).
```

Comment:
Similar to my comment on the help text, I think we should shuffle this around to be explicit and remove any potential ambiguity